contracts/GovHub.template

```diff
@@ -322,7 +322,7 @@ contract GovHub is System, IParamSubscriber {
 
   /// Remove a member
   /// @param member The address of the member to remove
-  function removeMember(address member) external onlyInit onlyGov onlyIfMember(member) {
+  function removeMember(address member) external onlyInit onlyGov onlyIfMember(member) { //@openissue
```

Comment:
voting a member out is very tricky and in some organizations illegal altogether.
was there a thought process behind this? is it documented anywhere? 
should't we consider, at minimum, a special quorum here?

Reply:
Not sure of the original thought process, but I think any long-term gov set would need this functionality to deal with 1-off malicious actors. If a majority of the gov members are acting in bad faith, it'll be hard to stop them. For example even with a quorum requirement for `removeMember`, the majority could just `addMember` a bunch of additional addresses.

Reply:
In my opinion, the current system is heavily predicated on the good faith of the majority. If we interrogate that principle then we'll have to make several changes to how things work, not just here in `removeMember`.

Therefore I think this would be better addressed in a more far-reaching governance overhaul, which is something we're tentatively looking at doing in the coming months.